addons/mail/models/mail_notification.py

```diff
@@ -102,3 +103,47 @@ def _notification_format(self):
             'failure_type': notif.failure_type,
             'partner_id': [notif.res_partner_id.id, notif.res_partner_id.display_name],
         } for notif in self]
+
+    @api.model
+    def cron_notify_admins(self):
```

Comment:
Please make this method private, as it directly uses sudo() without any check. 
```suggestion
    def _cron_notify_admins(self):
```